src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 30#{$units} 10#{$units} 10#{$units};
```

Comment:
The `padding` is affecting the size of the date-block width.

Reply:
Noted Thank you have made some changes please check again.